task_12/gulpfile.js

```diff
@@ -0,0 +1,46 @@
+var gulp = require('gulp');
+var sass = require('gulp-sass');
+var cleanCSS = require('gulp-clean-css');
+var imagemin = require('gulp-imagemin');
+var concat = require('gulp-concat');
+var uglify = require('gulp-uglify');
+const autoprefixer = require('gulp-autoprefixer');
+
+gulp.task('sass', function () {
+    return gulp.src('src/css/*.scss')
+        .pipe(sass())
+        .pipe(cleanCSS({compatibility: 'ie8'}))
+        .pipe(autoprefixer({
+            browsers: ['last 2 versions'],
+            cascade: false
+        }))
+        .pipe(gulp.dest('dist/css'))
+});
+
+gulp.task('img', function () {
+    return gulp.src('src/img/*')
+        .pipe(imagemin())
+        .pipe(gulp.dest('dist/img'));
+});
+
+
+gulp.task('concat', function () {
+    return gulp.src([
+        'src/js/loader.js',
```

Comment:
Do not use array if you work only with one file. It makes readability worst.

Reply:
If we look at in terms of scalability, do so will not be better?

In fact, this block is not needed right now.
And further I planned to understand in webPack and remove gulp